src/less/components/_data.less

```diff
@@ -63,7 +63,17 @@
 // Markup:
 //<p class="statusDataValue">
 //  <span class="active">
-//    Active
+//    Active Label
+//  </span>
+//</p> <br />
+//<p class="statusDataValue">
```

Comment:
@jeremytrawls people where confused, that we used two versions of displaying color, inside cards. Do you have any concerns with removing "statusDataValue"?